boltdesignsystem/bolt
# CC-472 Add reply prop and rename more_info prop to description in Listing Teaser

## Jira

https://pegadigitalit.atlassian.net/browse/CC-472

## Summary

Adds a `reply` prop, and rename `more_info` to `description` in the Listing Teaser component

## Details

1. Added `reply` prop to showcase a relevant reply.
2. Updated CSS to style the reply content.
3. Added docs to demonstrate the `reply` prop.
4. `more_info` has been renamed to `description`, its type has been changed to `any`.
5. `warning` prop's type has been changed to `any`.

## How to test

Run the branch locally and read the Listing Teaser props, especially the one with reply prop and Collaboration Center use case. Make sure the prop works as expected.
